core/src/main/java/org/mapfish/print/http/HttpRequestCache.java

```diff
@@ -54,16 +54,10 @@ public CachedClientHttpResponse(final ClientHttpResponse originalResponse) throw
             this.status = originalResponse.getRawStatusCode();
             this.statusText = originalResponse.getStatusText();
             this.cachedFile = File.createTempFile("cacheduri", null, HttpRequestCache.this.temporaryDirectory);
-            InputStream is = originalResponse.getBody();
-            try {
-                OutputStream os = new FileOutputStream(this.cachedFile);
-                try {
+            try (InputStream is = originalResponse.getBody()) {
```

Comment:
It does the same as the with python statement?

Reply:
Yes